tts_webui_extension/xtts_ft_demo/main.py

```diff
@@ -209,8 +236,11 @@ def preprocess_dataset(
                         audio_path,
                         target_language=language,
                         out_path=out_path,
-                        gradio_progress=progress,
+                        gradio_progress=None,
                     )
+                    # Автоматически фиксируем формат после препроцессинга
```

Comment:
```suggestion
                    # Automatically fix the format after preprocessing
```

Keep comments in  English